### ISSUE TITLE
Calibre cannot read and convert the file

### ISSUE THREAD

**Issue by gabbo83** [apprenticeharper/DeDRM_tools#667]
Hi ApprenticeHarper, hope you could help me.

I have latest Calibre version on  Windows 10, and installed DeDRM plugin for importating books from Kindle Windows Application  (v. 1.24.3).
I attach the report from Calibre Debug Mode.

Thanks!
[tmpy1x7qd.txt](https://github.com/apprenticeharper/DeDRM_tools/files/2529290/tmpy1x7qd.txt)

**Comment by gabbo83** [comment]
Thanks!
I understood that KFX input plugin was preinstalled in the latest version of DeDRM.

**Comment by ElleKayEm** [comment]
No, two separate plugins.  The latest DeDRM plugin and the KFX plugin are both needed.
